Podcast/PlayerControlsView.swift

```diff
@@ -0,0 +1,145 @@
+        addSubview(playPauseButton)
+        
+        forwardsButton = UIButton(frame: .zero)
+        forwardsButton.setBackgroundImage(#imageLiteral(resourceName: "skip_forward_icon"), for: .normal)
+        forwardsButton.adjustsImageWhenHighlighted = false
+        forwardsButton.addTarget(self, action: #selector(forwardButtonPress), for: .touchUpInside)
+        forwardsButton.frame = CGRect(x: 0, y: 0, width: forwardsBackwardsButtonSize, height: forwardsBackwardsButtonSize)
+        forwardsButton.center = CGPoint(x: playPauseButton.frame.maxX + controlButtonsCenterHorizontalOffset, y: controlButtonsCenterY)
+        addSubview(forwardsButton)
+        
+        forwardsLabel = UILabel(frame: .zero)
+        forwardsLabel.text = "30"
+        forwardsLabel.sizeToFit()
+        forwardsLabel.center = forwardsButton.center
+        addSubview(forwardsLabel)
+        
+        backwardsButton = UIButton(frame: .zero)
+        backwardsButton.setBackgroundImage(#imageLiteral(resourceName: "skip_backward_icon"), for: .normal)
+        backwardsButton.adjustsImageWhenHighlighted = false
+        backwardsButton.addTarget(self, action: #selector(backwardButtonPress), for: .touchUpInside)
+        backwardsButton.frame = CGRect(x: 0, y: 0, width: forwardsBackwardsButtonSize, height: forwardsBackwardsButtonSize)
+        backwardsButton.center = CGPoint(x: playPauseButton.frame.minX - controlButtonsCenterHorizontalOffset, y: controlButtonsCenterY)
+        addSubview(backwardsButton)
+        
+        backwardsLabel = UILabel(frame: .zero)
+        backwardsLabel.text = "30"
+        backwardsLabel.sizeToFit()
+        backwardsLabel.center = backwardsButton.center
+        addSubview(backwardsLabel)
+        
+        updateUI()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func updateUI() {
+        if Player.sharedInstance.isPlaying {
+            playPauseButton.setBackgroundImage(#imageLiteral(resourceName: "pause_icon"), for: .normal)
+        } else {
+            playPauseButton.setBackgroundImage(#imageLiteral(resourceName: "play_icon"), for: .normal)
+        }
+        
+        leftTimeLabel.text = Player.sharedInstance.currentItemElapsedTime().descriptionText
+        rightTimeLabel.text = Player.sharedInstance.currentItemRemainingTime().descriptionText
+        leftTimeLabel.sizeToFit()
+        rightTimeLabel.sizeToFit()
+        
+        leftTimeLabel.center = CGPoint(x: timeLabelHorizontalInset + leftTimeLabel.frame.width/2, y: sliderCenterY)
+        rightTimeLabel.center = CGPoint(x: frame.width - rightTimeLabel.frame.width/2 - timeLabelHorizontalInset, y: sliderCenterY)
+        
+        if !Player.sharedInstance.scrubbing {
+            slider.value = Float(Player.sharedInstance.getProgress())
+        }
+    }
+    
+    func playPauseButtonPress() {
+        Player.sharedInstance.togglePlaying()
```

Comment:
view should not call model methods, use a delegate protocol. EpisodeVC should call this